src/java_tools/import_deps_checker/javatests/com/google/devtools/build/importdeps/BUILD

```diff
@@ -96,10 +96,10 @@ java_test(
         "//src/java_tools/import_deps_checker/javatests/com/google/devtools/build/importdeps/testdata:testdata_lib_LibraryInterface",
         "//src/java_tools/import_deps_checker/javatests/com/google/devtools/build/importdeps/testdata:testdata_lib_LibraryModuleInfo",
         "//src/java_tools/import_deps_checker/javatests/com/google/devtools/build/importdeps/testdata:testdata_lib_Library_no_members",
-        "@local_jdk//:jre/lib/rt.jar",
+        "@remote_jdk8_linux//:jre/lib/rt.jar",
```

Comment:
Another option here would be to use the jar produced by DumpPlatformClassPath. We can consider that some day if we want to avoid pulling in JDK 8 just for this test.

Reply:
Good idea, pushed a new commit as this makes the diff much smaller.